web_src/js/ws.js

```diff
@@ -0,0 +1,512 @@
+/*
```

Comment:
Thank you for this PR. Can this htmx extension be imported via an external library instead of having the code directly in this codebase?

Reply:
found a fix by adding adjusting some webpack config